components/desktop/gnome2/gnome-media/Makefile

```diff
@@ -64,4 +65,4 @@ test:		$(NO_TESTS)
 
 BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
 
-include $(WS_TOP)/make-rules/depend.mk
```

Comment:
Do you mean from all of the files, or just this one?  I see five Makefiles that contain at least one of these lines.  Will this removal facilitate building `gmake REQUIRED_PACKAGES'?